src/qml/pages/MainPage.qml

```diff
@@ -61,13 +61,16 @@ Page {
                     }
 
                     Text {
+                        id: messageBody
                         color: Theme.secondaryTextColor
                         font.pointSize: UiHelper.fixFontSz(12)
-                        text: model.lastMessageBody
                         width: parent.width
+                        text: model.lastMessageBody
                         elide: Text.ElideRight
-                        textFormat: Text.RichText
+//                        textFormat: Text.RichText
+                        maximumLineCount: 1
```

Comment:
I don't know why this was added. Works fine without it for me.
For multi-line messages Qt replaces \n with space, it's fine.
Remove if you don't have the case there is a difference

Reply:
Removed